src/pages/views/explorer/explorerHeader.tsx

```diff
@@ -127,6 +135,9 @@ class ExplorerHeaderBase extends React.Component<ExplorerHeaderProps> {
     if (isGcpAvailable(gcpProviders, gcpProvidersFetchStatus, userAccess)) {
       return PerspectiveType.gcp;
     }
+    if (isIbmAvailable(ibmProviders, ibmProvidersFetchStatus, userAccess)) {
+      return PerspectiveType.gcp;
```

Comment:
Not gcp here. This should be PerspectiveType.ibm

Reply:
done